src/main/java/gale/Deadline.java

```diff
@@ -0,0 +1,42 @@
+package gale;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a deadline task that has a due date and time.
+ *
+ * @author kaikquah
+ */
+public class Deadline extends Task {
```

Comment:
Would it be better to suffix all your task files with "-Task"? For example, "DeadlineTask", or "ToDoTask" so your files are clearly separated?